Automattic/wp-calypso
# Purchases: Move loading placholder closer to Purchases codebase

On @mtias request I moved `<LoadingPlaceholder />` out of `/client/components/` closer to the code that is using it, i.e.: `/client/me/purchases/components/`.

### Testing
1. Go to http://calypso.localhost:3000/purchases.
2. Select a purchase that can be refunded. 
3. Go to Edit Payment Method page. Loader should work properly.
4. Go back.
5. Go to Cancel & Refund Purchase page. Loader should work properly.
4. Click Cancel & Refund Purchase button. Loader should work properly.

/cc @scruffian
